samples/Basic.Tests/TestHost.cs

```diff
@@ -22,8 +22,8 @@ private IServiceProvider CreateProvider()
 		{
 			var services = new ServiceCollection();
 			services.AddLogging();
-			services
-				.AddIdentity<AppUser, AppRole>()
+			services.AddIdentityCore<AppUser>(_ => { })
+				.AddRoles<AppRole>()
 				.AddUserStore<InMemoryUserStore<AppUser, AppRole>>()
```

Comment:
Ok so. It's true that we don't need to depend on anything besides the Core package. But I guess apps will still probably use `.AddIdentity<AppUser, AppRole>()`, right? So the samples will stay the same. And because our main project now drops the "Microsoft.AspNetCore.Identity" dependency, we should just add it in this sample instead, and revert this change.